megamek/src/megamek/server/Server.java

```diff
@@ -2366,6 +2376,8 @@ public boolean accept(Entity entity) {
             case PHASE_PHYSICAL:
             case PHASE_TARGETING:
             case PHASE_OFFBOARD:
+                deployOffBoardEntities();
```

Comment:
Is this going to loop through every entity in every phase?

Reply:
Yes, but only take action on offboard entities.